app/assets/javascripts/kassi.js

```diff
@@ -670,17 +670,20 @@ function initialize_update_account_info_form(locale, change_text, cancel_text, e
       $("#person_email_attributes_send_notifications").attr("disabled", "disabled");
     }
   );
-  $('#account_password_link').toggle(
-    function() {
+  var changeLinkVisible = true;
+  $('#account_password_link').click(function() {
+    if (changeLinkVisible) {
       $('#account_password_form').show();
       $(this).text(cancel_text);
       $('#person_password').focus();
-    },
-    function() {
+    } else {
       $('#account_password_form').hide();
       $(this).text(change_text);
     }
-  );
+
+    changeLinkVisible = !changeLinkVisible;
+  });
```

Comment:
There is, but it does totally different thing.

The old `toggle( fn, fn2, fn3, ... )` run the `fn` on the first click, `fn2` on the second, `fn3, on third and so on. The new `toggle` is only visual show/hide thing.